packages/interface/src/store/defaults.js

```diff
@@ -0,0 +1,14 @@
+export const DEFAULTS = {
+	areaControl: {
+		singleActiveAreas: {
```

Comment:
Can you explain what's the difference between `singleActiveAreas` and `multipleActiveAreas `? Can't it be merged into one object?

Reply:
The logic for singleActiveAreas ensures that only one area is kept active, when we enable an area the other area becomes disable.
multipleActiveAreas allows multiple active areas in a given zone, when we enable an area the other areas that were active are kept active.

Reply:
Is `multipleActiveAreas` used only for pinned items? If it's the case then it probably can be simplified as pinned items should mix shortcuts to all types of plugins for a given page type `edit-site` vs `edit-post` vs `edit-widgets`.

Reply:
For now, multipleActiveAreas is only used for pinned items but my plan was to use it for other things, like store which panels are open.